packages/@react-types/shared/src/index.d.ts

```diff
@@ -10,6 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
+/// <reference path="css.d.ts" />
```

Comment:
What happens when the components get published? Does this end up being loaded in user apps?